bin/crew

```diff
@@ -4,9 +4,6 @@ require_relative '../lib/color'
 # Disallow sudo
 abort 'Chromebrew should not be run as root.'.lightred if Process.uid == 0
 
-require 'find'
-require 'net/http'
-require 'uri'
```

Comment:
Good call.  Less is more.

Reply:
You can also take out `securerandom` as we don't need it anymore.

Reply:
> You can also take out `securerandom` as we don't need it anymore.

Done